Library/src/Multiplayer/Components/CinematicCameraSpaceComponent.cpp

```diff
@@ -0,0 +1,227 @@
+{
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::Position)]	= csp::common::Vector3::Zero();
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::Rotation)]	= csp::common::Vector4::Identity();
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::IsEnabled)]	= true;
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::FocalLength)] = 0.035f;
+	// 16:9
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::AspectRatio)] = 1.778f;
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::SensorSize)]	= csp::common::Vector2 {0.036f, 0.024f};
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::NearClip)]	= 0.1f;
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::FarClip)]		= 20000.0f;
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::Iso)]			= 400.0f;
+	// 60 FPS
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::ShutterSpeed)]		   = 0.0167f;
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::Aperture)]			   = 4.0f;
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::IsViewerCamera)]		   = false;
+	Properties[static_cast<uint32_t>(CinematicCameraPropertyKeys::ThirdPartyComponentRef)] = "";
+
+
+	SetScriptInterface(CSP_NEW CinematicCameraSpaceComponentScriptInterface(this));
+}
+
+float CinematicCameraSpaceComponent::GetFov()
+{
+	return 2.0f * atan(GetSensorSize().X / 2.0f * GetFocalLength()) * 1000.0f;
+}
+
+// transforms
+const csp::common::Vector3& CinematicCameraSpaceComponent::GetPosition() const
+{
+	return GetVector3Property(static_cast<uint32_t>(CinematicCameraPropertyKeys::Position));
+}
+
+void CinematicCameraSpaceComponent::SetPosition(const csp::common::Vector3& Value)
+{
+	SetProperty(static_cast<uint32_t>(CinematicCameraPropertyKeys::Position), Value);
+}
+
+const csp::common::Vector4& CinematicCameraSpaceComponent::GetRotation() const
+{
+	return GetVector4Property(static_cast<uint32_t>(CinematicCameraPropertyKeys::Rotation));
+}
+
+void CinematicCameraSpaceComponent::SetRotation(const csp::common::Vector4& Value)
+{
+	SetProperty(static_cast<uint32_t>(CinematicCameraPropertyKeys::Rotation), Value);
+}
+
+const csp::common::Vector3& CinematicCameraSpaceComponent::GetScale() const
+{
+	return csp::common::Vector3::One();
+}
+
+void CinematicCameraSpaceComponent::SetScale(const csp::common::Vector3& Value)
+{
+	// Ignored
+}
+
+
+/* ITransformComponent */
+
+
+
+SpaceTransform CinematicCameraSpaceComponent::GetTransform() const
+{
+	SpaceTransform Transform;
+	Transform.Position = GetPosition();
+	Transform.Rotation = GetRotation();
+	Transform.Scale	   = csp::common::Vector3::One();
+
+	return Transform;
+}
+
+void CinematicCameraSpaceComponent::SetTransform(const SpaceTransform& InValue)
+{
+	SetPosition(InValue.Position);
+	SetRotation(InValue.Rotation);
+}
+
+// FocalLength
```

Comment:
Space between focal and length?

Or remove space from some others to be consistent? (more comments below)

Reply:
I was in two minds here I think the point was it was supposed to be the property name (otherwise its hard to scan the file).  I'll make it normal english